### DISCUSSION TITLE
Support custom domain?

### DISCUSSION THREAD

**Discussion by wwhai** [community/community#3267] [Discussions]
Support custom domain at current version?

**Comment by evi-liu** [comment]
This space is currently reserved for feedback on Discussions and GitHub for Mobile. This is noted in our [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).
